src/components/Hero/HomepageHero.tsx

```diff
@@ -88,16 +88,16 @@ export const HomepageHero = () => {
   };
 
   return (
-    <Container width="full" bgColor="black" className="su-relative">
+    <Container width="full" className="su-relative su-bg-[hsla(214,39%,50%,1)]">
       <div className="">
         <m.div
           variants={prefersReduceMotion ? undefined : fadeVariants}
           initial="hidden"
           animate="visible"
         >
-          <Container bgColor="black" style={{ backgroundImage: darkMesh5 }} className="su-h-[34.3rem] md:su-h-[56.2rem] lg:su-h-[71.5rem] 2xl:su-h-[77.9rem] su-bg-no-repeat su-bg-[center_top_-2vw] 2xl:su-bg-top" />
+          <Container style={{ backgroundImage: skyGradient }} className="su-h-[34.3rem] md:su-h-[56.2rem] lg:su-h-[71.5rem] 2xl:su-h-[77.9rem] su-bg-no-repeat su-bg-[center_top_-2vw] 2xl:su-bg-top" />
         </m.div>
-        <div className="su-relative su-w-full su-bg-saa-black su--mb-1">
+        <div className="su-relative su-w-full su-bg-[hsla(214,39%,50%,1)] su--mb-1">
```

Comment:
This is all one off for now for demo-ing purposes. We'll name this color if we end up using this. Too early to tell.